src/components/home/elements/LeaseRecords.vue

```diff
@@ -276,4 +289,31 @@ export default {
     color: #696B8A;
     letter-spacing: 0;
 }
+.show-position {
+    position: absolute;
+    left: 13px;
+}
+.show-fee {
+    position: absolute;
+    right:240px;
+    width: 116px;
+    height: 36px;
+    background-color: #FFF;
```

Comment:
Remove "background-color". Let checkbox show the same background color with panel.